bigquery_etl/shredder/config.py

```diff
@@ -127,6 +128,10 @@ def fields(self) -> tuple[str, ...]:
     table="firefox_desktop_stable.quick_suggest_deletion_request_v1",
     field=QUICK_SUGGEST_CONTEXT_ID,
 )
+SEARCHWITH_SRC = DeleteSource(
+    table="firefox_desktop_stable.search_with_deletion_request_v1",
```

Comment:
This table doesn't exist. Did you want to use one of the [existing deletion request pings](https://dictionary.telemetry.mozilla.org/apps/firefox_desktop?itemType=pings&page=1&search=deletion&showUncollected=0)?

Reply:
I guess I just need the DeleteTarget then down below?

I'll admit this was all just cut and paste.